### PR TITLE
Feature/add favicon

### PR DESCRIPTION
**Pull request by raphaellueckl**
Adds a favicon. Works fully now.

### CODE REVIEW

**Comment by NoahLiechti** [comment]
I checked the branch out and it didn't work...am I doing something wrong?

**Comment by raphaellueckl** [comment]
@NoahLiechti It only works in production. Steps to make it work locally:

```
npm run build-storybook && npx http-server ./storybook-static
```

File: src/static/favicon.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...011-8.11551,12.1045H0v-7.949a5.9865,5.9865,0,0,1,.493-.4785c.386-.2835,18.213-22.4125,34.59-44.2335,14.717-19.0285,28.5255-37.5345,29.7365-39.297,2.9345-4.2625,7.163-13.4665,7.163-13.4665h21.782s.674,8.4665,1.309,10.522c.566,1.8165,13.8375,45.3615,14.15,45.8305l7.339-9.385-12.544-38.6135s-2.9395-7.2655-3.8965-8.354h25.4445a22.1658,22.1658,0,0,0,.83,6.9185c1.035,3.184,6.4895,22.8665,6.4895,22.8665s17.3535-21.753,18.369-23.3155a14.1,14.1,0,0,0,2.1435-6.4695h21.201s-3.8815,2.8365-10.6835,11.44c-2.285,2.8955-24.673,31.348-24.673,31.348s1.9535,6.66,2.9055,9.97552c.259.952.4395,1.5965.4395,1.665,0,.02948.493-.576,1.34252-1.665,5.77648-7.3195,32.051-41.7725,33.643-44.722,1.284-2.3825,3.1735-5.0925,4.282-8.0415H202.539s.4785,6.1765,1.1085,7.8855Zm-31.475-32.6125c-3.037,6.5335-20.913,28.296-20.913,28.296h28.3105s-5.488-16.8995-6.445-20.708a30.23266,30.23266,0,0,1-.557-7.4025C185.51748,169.6875,185.454,169.126,185.122,169.8485Zm-108.7745,0c-3.037,6.5335-20.913,28.296-20.913,28.296H83.745s-5.483-16.8995-6.44-20.708a30.23266,30.23266,0,0,1-.557-7.4025C76.748,169.6875,76.68,169.126,76.34749,169.8485ZM118.97,235.835l7.793-10.703c-.718-.77152-5.10748-14.082-5.10748-14.082l-7.53452,9.775Z"/></svg>
```
Comment (LucaMele) [inline src/static/favicon.svg]: Where do you use this file?

File: .storybook/manager-head.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!-- Favicons -->
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/site.webmanifest">
+<link rel="shortcut icon" href="/favicon.ico">
+<meta name="msapplication-TileColor" content="#da532c">
```
Comment (NoahLiechti) [inline .storybook/manager-head.html]: `TileColor`is set in `browserconfig.xml`. Isn't that enough?

File: src/static/favicons/site.webmanifest [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "",
-    "short_name": "",
+    "name": "Pattern Library",
```
[the remark this answers is not in the document]
Comment (raphaellueckl) [reply src/static/favicons/site.webmanifest]: Pattern

File: .storybook/manager-head.html (hunk 2 above)
Comment (raphaellueckl) [reply .storybook/manager-head.html]: These settings are the default, when you use the favicon generator, which is widely being used. I guess they know what they are doing, but I never checked that.

https://realfavicongenerator.net/

File: src/static/favicon.svg (hunk 1 above)
Comment (raphaellueckl) [reply src/static/favicon.svg]: Yeah sorry, I used it to create all the favicons. But you're correct, this is not being used.

File: .storybook/manager-head.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+<meta name="msapplication-config" content="/browserconfig.xml">
+<meta name="theme-color" content="#ffffff">
```
[the remark this answers is not in the document]
Comment (raphaellueckl) [reply .storybook/manager-head.html]: These settings are the default, when you use the favicon generator, which is widely being used. I guess they know what they are doing, but I never checked that.

https://realfavicongenerator.net/

File: src/static/favicon.svg (hunk 1 above)
Comment (raphaellueckl) [reply src/static/favicon.svg]: done

**Pull request merged**